internal/resource-request-operator/broadcaster.go

```diff
@@ -101,58 +124,100 @@ func (b *Broadcaster) onUpdate(oldObj, newObj interface{}) {
 		(newNode.Status.Phase == corev1.NodeTerminated || newNode.Status.Phase == corev1.NodePending) {
 		subResources(currentResources, oldNodeResources)
 	}
-	if err := b.writeResources(currentResources); err != nil {
-		klog.Errorf("OnUpdate error: unable to write allocatable of Node %s: %s", newNode.Name, err)
-	}
+	b.writeNodesResources(currentResources)
 }
 
 // react to a Node Delete.
-func (b *Broadcaster) onDelete(obj interface{}) {
+func (b *Broadcaster) onNodeDelete(obj interface{}) {
 	node := obj.(*corev1.Node)
 	toDelete := &node.Status.Allocatable
-	currentResources := b.allocatable.DeepCopy()
+	currentResources := b.readNodeResources()
 	if node.Status.Phase == corev1.NodeRunning {
 		subResources(currentResources, *toDelete)
-		if err := b.writeResources(currentResources); err != nil {
-			klog.Errorf("OnAdd error: unable to write allocatable of Node %s: %s", node.Name, err)
+		b.writeNodesResources(currentResources)
+	}
+}
+
+func (b *Broadcaster) onPodAdd(obj interface{}) {
+	pod := obj.(*corev1.Pod)
+	klog.V(2).Infof("OnPodAdd: Add for pod %s\n", pod.Name)
```

Comment:
Would personally raise to V(4) or V(5), as debug/trace level. The same for the one below